  <launch>
    <arg name="map_size_x" value="120.0"/>
    <arg name="map_size_y" value="40.0"/>
    <arg name="map_size_z" value=" 3.0"/>

    <arg name="odom_topic" value="visual_slam/odom" />

    <include file="$(find vrb_planner)/launch/run_in_sim.launch">
        <arg name="drone_id" value="3"/>

        <arg name="init_x" value="-38.0"/>
        <arg name="init_y" value="0.0"/>
        <arg name="init_z" value="0.5"/>

        <arg name="target_x" value="42.0"/>
        <arg name="target_y" value="0.0"/>
        <arg name="target_z" value="0.5"/>

        <arg name="map_size_x"              value="$(arg map_size_x)"/>
        <arg name="map_size_y"              value="$(arg map_size_y)"/>
        <arg name="map_size_z"              value="$(arg map_size_z)"/>
        <arg name="odom_topic"              value="$(arg odom_topic)"/>        
    </include>

</launch>